feat: support column_width in xlsx format (#516)
jazzband/tablib: docs/formats.rst

can set to a number of lines that should be skipped before starting to read
data.

The ``export_set()`` method supports a ``column_width`` parameter. Depending on the
value you pass, the column width will be set accordingly. It can be either ``None``, an integer, or "adaptive".
If "adaptive" is passed, the column width will be unique for every column and will be 
calculated based on values' length. For example:: 

    data = tablib.Dataset()
    data.export('xlsx', column_width='adaptive')

> .. versionchanged:: 3.3.0

Reviewer: 3.3.0 has been released, let's target 3.4.0:

```suggestion
.. versionchanged:: 3.4.0
```

can set to a number of lines that should be skipped before starting to read
data.

The ``export_set()`` method supports a ``column_width`` parameter. Depending on the

> value you pass, the column width will be set accordingly. It can be either None, int, or "adaptive".

Reviewer: ```suggestion
value you pass, the column width will be set accordingly. It can be either ``None``, an integer, or "adaptive".
```

If "adaptive" is passed, the column width will be unique for every column and will be 
calculated based on values' length. Example of usage

```python3
data = tablib.Dataset()
data.export('xlsx', column_width='adaptive')

> ```

Reviewer: This is RST not Markdown, so we can use a double colon at the end of the last line to activate the code block:

```suggestion
calculated based on values' length. For example::

    data = tablib.Dataset()
    data.export("xlsx", column_width="adaptive")

can set to a number of lines that should be skipped before starting to read
data.

> The ``export_set()`` method supports a ``column_width`` parameter. Depending 
> on the value passed, the column width will be set accordingly. It can be 
> either ``None``, an integer, or default "adaptive". If "adaptive" is passed, 
> the column width will be unique and will be calculated based on values' length. 
> For example:: 

Reviewer: Let's remove those trailing spaces:
```suggestion
The ``export_set()`` method supports a ``column_width`` parameter. Depending
on the value passed, the column width will be set accordingly. It can be
either ``None``, an integer, or default "adaptive". If "adaptive" is passed,
the column width will be unique and will be calculated based on values' length.
For example::
```
